### DISCUSSION TITLE
Import Offline Draft Results

### DISCUSSION THREAD

**Discussion by gary-roach** [mattdodge/yahoofantasy#59] [Ideas]
This is an awesome project!

I'm interested in importing my offline draft results into yahoo. I have some web development experience and I'd be happy to implement this.

The documentation on Yahoo's site is pretty poor. I assume this is available in the Yahoo API. Do you have a reference to the documentation for this?

Does this project have any wrappers for this already?

**Comment by mattdodge** [comment]
Unfortunately I haven't seen an API for importing offline draft results, despite this seeming like an obvious thing to have. Or at least a CSV upload or something. AFAIK you're stuck with manually entering draft results for now but I'd be happy to be proven wrong
